src/serialization/tokenizer.hpp

```diff
@@ -20,14 +20,17 @@
 
 #include "buffered_istream.hpp"
 
-#include <limits>
 #include <istream>
 #include <string>
 
 // use of illegal utf8 character for this purpose was added in a76be7ef1e921dabacd99f16ef440bf9673b8d98
 // has something to do with the result of the preprocessor whose format is essentially undocumented and I don't intend to delve into that as of writing this comment
 #define INLINED_PREPROCESS_DIRECTIVE_CHAR 254
 
+// normal ascii is 0-127
+// extended ascii is from 128-255, none of which need any special handling
+#define START_EXTENDED_ASCII 128
```

Comment:
Can't this be a constant instead of a macro?

Reply:
I will make them constexpr then.

Reply:
static const is the normal way (constexpr is fine too though).